pkg/virt-controller/watch/migration/migration.go

```diff
@@ -557,6 +567,9 @@ func (c *Controller) updateStatus(migration *virtv1.VirtualMachineInstanceMigrat
 
 	controller.SetVMIMigrationPhaseTransitionTimestamp(migration, migrationCopy)
 	controller.SetSourcePod(migrationCopy, vmi, c.podIndexer)
+	if err := c.setSynchronizationAddressStatus(migrationCopy); err != nil {
```

Comment:
Missing check for feature gate?

Reply:
Don't need it, if the feature gate is not enabled then the KubeVirt CR will not have the value, and thus this function will not do anything.

Reply:
Please see https://github.com/kubevirt/kubevirt/pull/14882/files#r2171267710